### DISCUSSION TITLE
Directus change field from text to long text

### DISCUSSION THREAD

**Discussion by moayadsquad** [directus/directus#12717] [Archive]
Directus does not support long text fields, so I make new migration file that change data type from text to long text, it's works, but when apply my snapshot file , the fields return from long text field to text, is it possible to use long text in directus?

**Comment by rijkvanzanten** [comment]
The schema snapshotting engine doesn't currently rely on the specific database type used, but relies on the "regular" field creation logic which relies on knex's abstractions for creating those types.
